mlflow/azureml/__init__.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import print_function
+
+import os
+import shutil
+
+import mlflow
+from mlflow import pyfunc
+from mlflow.models import Model
+from mlflow.tracking import _get_model_log_dir
+from mlflow.utils.file_utils import TempDir
+from mlflow.version import VERSION as mlflow_version
+
+
+def deploy(app_name, model_path, run_id, mlflow_home):
+    """Deploy MLflow model to Azure ML.
```

Comment:
Nit: I've preferred to keep the `"""` on its own line to align the text better in other docs